subprojects/docs/src/docs/userguide/troubleshooting/validation_problems.adoc

```diff
@@ -389,6 +389,7 @@ To fix this, look at the error message which indicates the list of possible solu
 Please find below the list of unsupported value type and links to respective sections of the user manual:
 
 `ResolvedArtifactResult`:: <<incremental_build#sec:task_input_using_dependency_resolution_results, Using dependency resolution results as task inputs>>
+`java.net.URL`:: <<incremental_build#sec:task_input_using_dependency_resolution_results, Using dependency resolution results as task inputs>>
```

Comment:
My vote is the user manual.

Reply:
This file is in the user manual, no? Do you mean to add the info to some other part?

Reply:
For `ResolvedArtifactResult`, we link to another section in the user manual that has more information. Do we want to do the same for `java.net.URL`? In this case we would need to put some information about why we do not allow the type as input and I'm not sure where that information should go. Does such content for other types exist already? Otherwise we could maybe add it at https://docs.gradle.org/current/userguide/incremental_build.html#sec:task_input_output_annotations ?

Reply:
Ah, I see, sorry. I would change the line above to:

```
The list of unsupported value types:
```

Leave `ResolvedArtifactResult` as is, and just explain why `URL` is unsupported here without linking anywhere. WDYT? We could even add a link to the JDK bug about `hashCode()` – https://bugs.openjdk.org/browse/JDK-8075619. The bug mentions that it won't be fixed, too.